src/main.rs

```diff
@@ -7,46 +7,87 @@ mod terminal;
 mod traits;
 mod ui;
 
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 
 use anyhow::Result;
 use application::Application;
 use libc::{EXIT_FAILURE, EXIT_SUCCESS};
-use log::{info, LevelFilter};
+use log::{info, warn, LevelFilter};
 use terminal::TerminalWrapper;
 
-fn init_logging() -> log2::Handle {
-    let log_dir = if let Ok(_dir) = std::env::var("XDG_RUNTIME_DIR") {
-        // store log in the current directory for convenience
-        // we use XDG_RUNTIME_DIR to detect the fact that we are running on desktop linux
-        PathBuf::from("./")
+const EVE_MONITOR_LOG_DIR_EVE: &str = "/persist/monitor/log";
+const EVE_MONITOR_LOG_DIR: &str = "./persist/monitor/log";
+
+fn get_base_log_dir() -> &'static str {
+    // we use XDG_RUNTIME_DIR to detect the fact that we are running on desktop linux
+    // FIXME: is there a better way?
+    if let Ok(_dir) = std::env::var("XDG_RUNTIME_DIR") {
```

Comment:
Do it compile time? Since we are compiling binary for specific platform and we can do it like this:

```rust
if cfg!(target_os = "linux") {
  // ...
} else {
  // ...
}
```

or does desktop linux means having x server running? Then there's also DISPLAY env variable (env::var("DISPLAY") )

Reply:
@uncleDecart I thought about checking "/persist" exists. this is true only for eve